lib/python/pyflyby/_modules.py

```diff
@@ -482,6 +480,7 @@ def containing(cls, identifier):
             result = module.module
         except Exception as e:
             raise ImportError(e)
+        assert False
```

Comment:
This seem to be never reached, as static typing tell us that next line is wrong.